staging/publishing/rules.yaml

```diff
@@ -1,3 +1,5 @@
+skip-source-branches:
+- release-1.10
```

Comment:
can we delete all the 1.10 rules?

Reply:
Yup. :+1: 

> final 1.10 release was cut and has been published. are these files net new?

Only the last commit is new.